.github/workflows/doc.yml

```diff
@@ -0,0 +1,18 @@
+name: doc
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'docs/**'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - run: pip install mkdocs-material
```

Comment:
Isn't the "i18n" plugin missing?

Reply:
Yes. Think I mixed up something in the final commit here https://github.com/opengisch/QgisModelBaker/pull/625/commits/1dbdbf5c09dc4ef778e82d9aa58992fcd322eda0 Remember that I tried something out with requirements etc.